harness/apisupport.harness/external/binaries-list

```diff
@@ -20,4 +20,5 @@
 # D4EF66C1CC8A5B3C97E0CC7C210227AAEC1F1086 jsearch-2.0_05.jar
 A806D99716C5E9441BFD8B401176FDDEFC673022 bindex-2.2.jar
 6FF4C411EE231AB8042C9047373F87FC64D0299D harness-launchers-8.2.zip
-B7F800780CC94C214B0FE9DA455ADCD20D976C15 https://archive.apache.org/dist/netbeans/native/netbeans-launchers/1-94a19f0/launcher-external-binaries-1-94a19f0.zip launcher-external-binaries-1-94a19f0.zip
+#DB781CBFDA3565BC55742D90E0C111204764D85C https://archive.apache.org/dist/netbeans/native/netbeans-launchers/2-6c17cc6/launcher-external-binaries-2-6c17cc6.zip launcher-external-binaries-2-6c17cc6.zip
+DB781CBFDA3565BC55742D90E0C111204764D85C https://mbien.dev/tmp/launcher-external-binaries-2-6c17cc6.zip launcher-external-binaries-2-6c17cc6.zip
```

Comment:
Are we sure of the `https://mbien.dev/tmp/`?

Reply:
the line above will be the final line once the release is done

Reply:
And for the vote it needs to be the link I posted above, once it's in the staging tree. This PR is part of the verification process for the vote.

Reply:
if someone wants to move the stuff around, i can update the PR tomorrow evening. The hash is in the PR so not much can go wrong while moving files.

Reply:
but my raspi 3 is honored to be temporary member of the ASF build cluster :)

Reply:
switched yesterday to the endorsed location